### ISSUE TITLE
[Bug Report][3.3.11] VTooltip text contrast does not meet WCAG standards for accessible text

### ISSUE THREAD

**Issue by ClemDee** [vuetifyjs/vuetify#17998]
### Environment
**Vuetify Version:** 3.3.11
**Vue Version:** 3.3.4
**Browsers:** Firefox 116.0
**OS:** Linux

### Steps to reproduce
- go to reprodution link
- open your browser accessibility panel
- look for contrast issues
- look for the tooltip text contrast entry

### Expected Behavior
Tooltip text is legible.
The text contrast is at least 4.5 : 1 (minimum ratio AA rating)

### Actual Behavior
Tooltip text is hard to read, and especially not legible on dark mode.
The text has contrast of 3.70 on light mode, and 3.07 on dark mode.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqlUsuO4jAQ/JWWLzASSTSaPSGYZX9hhfYymYNJGvDi2JbdyewK8e/btnkE5jLSciGuqu5yVfJ2FD+cK4cexVwsCDunJeFrbQAWQyGdS4/p0FhDUhn0ZyiBGzKwarRqDstakN3tNK732GEtriqAdcIhE1d08WttrSblbhDAUHS2Rc3Lwt5+XARiLJENqUGS9Sxy0qOhe57wD8XL8B8YS6BxpzaabzQWadtIUtawEE075qpbvCrluzTAp/sKIpIbWlSj5vgYGq8cQUDqc4Gqc9YTHIHsT9zCCbbedjDh2id3dB9yfWMFqe1fVkUd+wdee2sGlnnjdPoEy1cg3+PTTUhp1fK6dTrmbu+KFWk855ymqXKn7Ubq0kh+HqTuo+qOaXofu89k2Up/uPT2HSZa7faUosXfHCaRn7D9osrdsJmYiZy76KQrfwdr+BM8xpn6TIRazCEhEeMq4rkWeyIX5lXVtIbH+GtRgy8NUmVcV61YVvnekOqwaG23eilfym9VqwKN4RJDV2y8/QjoeUktZiOb2PgXrM7K5PD8nC3OWKHlJkSPT7srNhzQF9xdix79VyM9jI1jPVCfokX3U21OXLhW5hAeum5C6vntcsn/y5y2xVXv0fE0E+mibJ0nxPs/7IVyyg==)

### Other comments
Even though vuetify theming can be modified by anyone, its default configuration should at least pass basic accessibility standards.

I guess modifying `$tooltip-background-color` and `$tooltip-text-color` sass variables default values should suffice to solve the issue.
